src/sswsdk/interface.py

```diff
@@ -134,10 +134,15 @@ class DBInterface(object):
 
     db_map = dict()
 
-    def __init__(self):
+    def __init__(self, **kwargs):
 
         super(DBInterface, self).__init__()
 
+        # Store the arguments for redis client
+        self.redis_kwargs = kwargs
+        if len(self.redis_kwargs) == 0:
+            self.redis_kwargs['unix_socket_path'] = self.REDIS_UNIX_SOCKET_PATH
+
```

Comment:
Are you not going to do anything if the length not equal to 0?
Are you sure that 'unix_socket_path' exists in the parameter?